improver_tests/calibration/rainforests_calibration/test_ApplyRainForestsCalibrationLightGBM.py

```diff
@@ -232,12 +233,9 @@ def test__evaluate_probabilities(
     assert np.all(error_threshold_cube.data >= 0)
     assert np.all(error_threshold_cube.data <= 1)
     # check data is 1 where forecast + error < 0
-    precip_forecast_reshaped = np.reshape(
-        precip_forecast, error_threshold_cube.data.shape[1:]
-    )
+    precip_forecast_reshaped = np.reshape(forecast, error_threshold_cube.data.shape[1:])
```

Comment:
Would it not be simpler to just use `ensemble_forecast.data` rather than `precip_forecast_reshaped`?

Reply:
Fixed, thanks